draft-ietf-masque-h3-datagram.md

```diff
@@ -180,56 +161,58 @@ of {{QUIC}}):
 
 ~~~
 HTTP/3 Datagram {
-  Quarter Stream ID (i),
+  Encoded Stream ID (i),
```

Comment:
Since we're changing the wire format, we need to bump the value of the SETTING

Reply:
Done!